modules/peers.js

```diff
@@ -76,7 +79,7 @@ __private.countByFilter = function (filter, cb) {
  * @returns {setImmediateCallback|[Peer]} peers
  */
 __private.getByFilter = function (filter, cb) {
-	var allowedFields = ['ip', 'port', 'state', 'os', 'version', 'broadhash', 'height', 'nonce'];
+	var allowedFields = ['ip', 'port', 'httpPort', 'state', 'os', 'version', 'broadhash', 'height', 'nonce'];
```

Comment:
Do we should include `httpPort` into peer table and store it into database?

Reply:
I agree,  definitely should be included. There is an issue #855, are you ok with introducing those change in it?